### PR TITLE
Refactored 3.11 hack to be more readable

### PR DESCRIPTION
**Pull request by radtriste**
and added missing kogitoinfra crd file

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: hack/3.11deploy.sh [hunk 1]
```diff
@@ -13,7 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-for FILE in deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml deploy/crds/app.kiegroup.org_kogitodataindices_crd.yaml deploy/role.yaml deploy/service_account.yaml deploy/role_binding.yaml deploy/operator.yaml
+# Define CRD files
+crd_folder="deploy/crds"
+crd_files=("app.kiegroup.org_kogitoapps_crd.yaml" "app.kiegroup.org_kogitodataindices_crd.yaml" "app.kiegroup.org_kogitoinfras_crd.yaml")
```
Comment (ricardozanini) [inline hack/3.11deploy.sh]: Won’t be better to scan the directory for CRD files instead of having them fixed here?

Comment (radtriste) [reply hack/3.11deploy.sh]: We could but I found it more complex to code that's it XD

Comment (ricardozanini) [reply hack/3.11deploy.sh]: Just add a ==~ to compare regexp looking for “crd” pattern. Hahahah

Comment (radtriste) [reply hack/3.11deploy.sh]: updated :)

**Pull request merged**